airflow/config_templates/config.yml

```diff
@@ -1946,11 +1946,12 @@
       type: string
       example: ~
       default: "30"
-    - name: deactivate_stale_dags_interval
+    - name: parsing_cleanup_interval
```

Comment:
Do we need `newsfragment` entry for this?

Reply:
Wouldn't hurt, but it's not a big thing as the old one will still work and you'll get warnings in every component for using the old value